plugins/com.google.cloud.tools.eclipse.appengine.standard.java8/build.properties

```diff
@@ -1,7 +1,7 @@
-source.. = src/,\
-           src-test/
```

Comment:
This will prevent `AppEngineXsltTransformTest.java` from being included in the final build, right? Also what's the criterion to put the test class in the main bundle while put other classes in the `.test` bundle?

Also I noticed `output.. = bin/` instead of `output.. = target/classes` in both the main bundle and the `.test` bundle, and unlike other bundles, the `target` folder doesn't have OGSi stuff. (Strangely, I don't see the `bin` folder though.) Is this normal?

Lastly, the following files which are not checked into the repo are auto-generated when I import the `java8` bundles.
```
com.google.cloud.tools.eclipse.appengine.standard.java8.test/.settings/org.eclipse.core.resources.prefs
com.google.cloud.tools.eclipse.appengine.standard.java8.test/.settings/org.eclipse.m2e.core.prefs
```